stories/module-search-console-components.stories.js

```diff
@@ -2336,3 +2337,112 @@ generateReportBasedWidgetStories( {
 	wrapWidget: false,
 	setup,
 } );
+
+generateReportBasedWidgetStories( {
+	moduleSlugs: [ 'search-console' ],
+	datastore: STORE_NAME,
+	group: 'Search Console Module/Components/Module Page/ Popular Pages Widget',
```

Comment:
+1 for also adding a story!

@eugene-manuilov Could you maybe do this as a follow-up to #2067 and #2068 as well?